language-design.md

```diff
@@ -106,7 +138,9 @@ you do so? If not, why not?
 
 **Response**
 
+Intuitiveness is the underlying feature of good design, whether talking about a DSL, API, or any other product that will be used by the general masses. A quality intuitive product will work with minimal instructions and will enable non-experts to relate without a challenging barrier of entry. Therefore, it is important to note that the assumption of interfacing with natural language is inherently _good_ is _false_. Specifically, while it is true natural language is relatable to everyone, not just programmers, it is also important to remember a programming language cannot be ambiguous. The challenge highlighted in these seemingly contrary claims, however, is when does natural language stop being beneficial for a programming language’s intuitiveness. While all users can mostly relate to natural language, computers, unfortunately cannot. A program, in its most basic form, is a series of formal specifications that define a computer’s behavior. Unlike human language, a good programming language _cannot_ enable ambiguities and _should_ be consistent. Since a programming language needs to be formal and precise, by relying too heavily on natural language, a designer can accidentally trick users into thinking a language is more similar to natural language than it really is, thus hurting intuition. This seems to be the issue eluded to in Cook’s comment that the “main problem is that AppleScript only appears to be a natural language” [Cook, 2007, 1-20]. By making AppleScript be too much like natural language, the users end up hindered by erroneously equating the distinct idiosyncrasies of natural language and AppleScript. Thus, while relying on natural language to inform intuition, relying too heavily on it may ultimately induce more confusion than it relieves.
 
+Therefore, if given the option to incorporate natural language into a DSL, I would likely include some, but do so moderately. Like all things, too much of a good thing can quickly become bad. To avoid creating a language that is “easy to read… but quite hard to write…” [Cook 2007, page 1-20] like AppleScript, I would want to avoid relying too heavily on natural language. Therefore, by using a nice compromise, like Python’s _in_ operator (in contrast to the contains() function of other languages), a programming language can simultaneously improve readability and build a user’s intuition while also keeping the boundary between natural languages and programming languages discrete.
```

Comment:
I would argue that intuitiveness really depends on the perspective of the user. If your user is someone who has little to no experience with programming languages, the _in_ operator would probably be the most intuitive choice. However, if the language is situated in a more technical context and the ideal user would have experience with more complex languages, it might be more intuitive to the user for the language to use the contains() function.